test/unittests/eof_validation_test.cpp

```diff
@@ -383,12 +386,8 @@ TEST_F(eof_validation, EOF1_truncated_push)
             add_test_case(container, EOFValidationError::truncated_instruction);
         }
 
-        const bytes code{opcode + bytes(required_bytes, 0) + uint8_t{OP_STOP}};
-        code_size_byte = static_cast<uint8_t>(code.size());
-
-        eof_header[18] = static_cast<uint8_t>(instr::traits[opcode].stack_height_change);
-
-        const auto container = eof_header + code;
+        const auto container = eof_bytecode(push(bytes(required_bytes, 0)) + OP_STOP,
```

Comment:
I guess this could also stay like
```suggestion
        const auto container = eof_bytecode(opcode + bytes(required_bytes, 0) + OP_STOP,
```
maybe more clear what's happening this way